alert-policies/apache-traffic-server/FreeCacheRAM.yml

```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: Free Cache RAM Percent
+
+# Description and details
+description: |+
+  This alert is triggered if the cache RAM free lower than 5% for 5 minutes.
```

Comment:
```suggestion
  This alert is triggered if the free space in RAM memory cache is below 10% for 5 minutes.
```

Reply:
Hi @mdumpati,
As per your suggestions, I have made changes. Kindly review it.

Thanks.